lib/services/runtime/programs/resume.ts

```diff
@@ -14,14 +14,15 @@ export enum ResumeVariables {
   FOLLOW_VOICE = '__voice1__',
 }
 
-export const promptToSSML = (content = '', voice: string | undefined) => {
-  if (!voice || voice === 'Alexa' || !content) {
-    return content;
+export const promptToSSML = (content: string | undefined, voice: string | undefined) => {
```

Comment:
can we make voice optional, this should fix lint issues without any code refactoring, isn't it?
```suggestion
export const promptToSSML = (content: string = '', voice? string) => {
```

Reply:
the same issue will be present, the linter only wants one """optional""" argument (`x: T = y`) which must be the final argument